tweets/templatetags/tweet_tags.py

```diff
@@ -4,8 +4,24 @@
 
 @register.filter
 def can_edit_tweet(user, tweet):
-  return user.id == tweet.user_id and tweet.original_tweet is None
+  if user.id != tweet.user_id:
+    return False
+  if tweet.original_tweet_id is not None:
+    return False
+  return True
+  # return user.id == tweet.user_id and tweet.original_tweet_id is None
 
 @register.filter
 def can_delete_tweet(user, tweet):
   return user.id == tweet.user_id
+
+@register.filter
+def can_retweet(user, tweet):
+  if tweet.original_tweet_id is None:
+    return False
```

Comment:
If `original_tweet_id` is None, means its an original tweet. We should be able to retweet.